pkg/statistics/handle/initstats/load_stats_page.go

```diff
@@ -57,6 +101,10 @@ func (ls *RangeWorker) loadStats() {
 		if err := ls.dealFunc(task); err != nil {
 			logutil.BgLogger().Error("load stats failed", zap.Error(err))
 		}
+		if ls.logger != nil {
+			completeTaskCnt := ls.completeTaskCnt.Add(1)
+			ls.logger.Info(fmt.Sprintf("load %s [%d/%d]", ls.taskName, completeTaskCnt, ls.taskCnt))
```

Comment:
seems it's sampling output
1%    50%   90%   100% 
if 100% is missed，will the log msg looks wired?

Reply:
If it is complete, It must print ```complete```.

Reply:
load xxxx [completed task count / total task count]